demo/k8s-validating-admission-policy/k8srequiredlabels_template.yaml

```diff
@@ -28,14 +28,8 @@ spec:
       code:
         - engine: K8sNativeValidation
           source:
-           apiVersion: admissionregistration.k8s.io/v1alpha1
-           kind: ValidatingAdmissionPolicy
-           spec:
-             paramKind:
-               apiVersion: "foobar/v1"
-               kind: "Placeholder"
-             validations:
-             - expression: "params.spec.parameters.labels.all(entry, has(object.metadata.labels) && entry.key in object.metadata.labels)"
-               messageExpression: '"missing required label, requires all of: " + params.spec.parameters.labels.map(entry, entry.key).join(", ")'
-             - expression: "!params.spec.parameters.labels.exists(entry, has(object.metadata.labels) && entry.key in object.metadata.labels && !string(object.metadata.labels[entry.key]).matches(string(entry.allowedRegex)))"
-               message: "regex mismatch"
+            validations:
```

Comment:
This is going to break with 3.13.0 release right? How about updating the readme in this folder to build from head or use `openpolicyagent/gatekeeper:dev`?

Reply:
Ah, good point.

Reply:
updated